### PR TITLE
Connect ceremony details to contentful natalie

### PR DESCRIPTION
**Pull request by burtonjong**

### CODE REVIEW

**Comment by burtonjong** [comment]
rebased off #120 i believe

File: src/app/contentfulTypes.ts
Comment (justin-phxm) [inline src/app/contentfulTypes.ts:1]: I don't believe we need to create a new contentful type to do this. Lets just add an extra field onto the HackathonDetails Type.

File: src/app/actions.ts
Comment (justin-phxm) [inline src/app/actions.ts:1]: I have an upgrade in #147 that will prevent the need to do this

File: src/components/Dashboard/ImportantInformation.tsx
Comment (justin-phxm) [inline src/components/Dashboard/ImportantInformation.tsx:1]: an upgrade in #147 will fix this. css changes?

File: src/components/Dashboard/JudgingInfo.tsx [hunk 1]
```diff
@@ -1,16 +1,116 @@
...
 import JudgeIcon from "@/images/dashboard/JudgeIcon.png";
+import { useQuery } from "@tanstack/react-query";
 
+import { useUser } from "../contexts/UserContext";
 import Card from "./Card";
 
+const client = generateClient<Schema>();
+
 export default function JudgingInfo() {
   const href = "#";
```
Comment (justin-phxm) [inline src/components/Dashboard/JudgingInfo.tsx:17]: Do we want to leave this href blank?

File: src/components/Dashboard/JudgingInfo.tsx
Comment (justin-phxm) [inline src/components/Dashboard/JudgingInfo.tsx:1]: We should take advantage of NextJS SSR by breaking this component down into separate smaller components.

Here are some example of me using it:
https://github.com/Code-the-Change-YYC/hackathon-management-tool/blob/74c388e4859b30bfdfdd1cce27cc5b60ebb97b77/src/components/Dashboard/TotalTeams.tsx#L8

https://github.com/Code-the-Change-YYC/hackathon-management-tool/blob/74c388e4859b30bfdfdd1cce27cc5b60ebb97b77/src/components/Dashboard/TotalParticipants.tsx#L8

https://github.com/Code-the-Change-YYC/hackathon-management-tool/blob/74c388e4859b30bfdfdd1cce27cc5b60ebb97b77/src/components/Dashboard/NumFoodTickets.tsx#L11-L12

Comment (justin-phxm) [reply src/components/Dashboard/JudgingInfo.tsx:1]: Also if we turn this to a server component, consider wrapping in a SuspenseWrapper.

https://github.com/Code-the-Change-YYC/hackathon-management-tool/blob/74c388e4859b30bfdfdd1cce27cc5b60ebb97b77/src/components/SuspenseWrapper.tsx#L5

**Review by justin-phxm — CHANGES_REQUESTED** [review]
Lots of good stuff but maybe we can wait until we get my PRs in before continuing on this one. There is a considerable amount of overlap and optimizations that can still be done.

File: src/components/Dashboard/JudgingInfo.tsx
Comment (burtonjong) [reply src/components/Dashboard/JudgingInfo.tsx:1]: Make a ticket / issue and a recruit or someone else can finish this later as its low prio

**Review by ideen1 — APPROVED** [review]
LGTM

**Pull request merged**